Sources/Swift/Tools/SentryViewPhotographer.swift

```diff
@@ -37,60 +37,76 @@ class SentryViewPhotographer: NSObject, SentryViewScreenshotProvider {
         self.redactBuilder = UIRedactBuilder(options: redactOptions)
     }
     
-    func image(view: UIView, onComplete: @escaping ScreenshotCallback ) {
+    func image(view: UIView, onComplete: @escaping ScreenshotCallback) {
         let redact = redactBuilder.redactRegionsFor(view: view)
         let image = renderer.render(view: view)
+        let viewSize = view.bounds.size
         
-        let imageSize = view.bounds.size
         dispatchQueue.dispatchAsync {
-            let screenshot = UIGraphicsImageRenderer(size: imageSize, format: .init(for: .init(displayScale: 1))).image { context in
-                
-                let clipOutPath = CGMutablePath(rect: CGRect(origin: .zero, size: imageSize), transform: nil)
-                var clipPaths = [CGPath]()
+            let screenshot = self.maskScreenshot(screenshot: image, size: viewSize, masking: redact)
+            onComplete(screenshot)
+        }
+    }
+    
+    func image(view: UIView) -> UIImage {
+        let redact = redactBuilder.redactRegionsFor(view: view)
+        let image = renderer.render(view: view)
+        let viewSize = view.bounds.size
+        
+        return self.maskScreenshot(screenshot: image, size: viewSize, masking: redact)
+    }
+    
+    private func maskScreenshot(screenshot image: UIImage, size: CGSize, masking: [RedactRegion]) -> UIImage {
```

Comment:
`m`: I'm a bit surprised that we don't need to update any tests for this change here.

Reply:
Because the behavior didn't changed.